### PR TITLE
make authorize_call free and operational

### LINKED ISSUE #608: Make trigger_call operational and not paying fees

### PR DESCRIPTION
**Pull request by ermalkaleci**
Closes #608

### CODE REVIEW

File: authority/src/lib.rs [hunk 1]
```diff
@@ -342,7 +342,11 @@ pub mod module {
 			Ok(())
 		}
 
-		#[pallet::weight(T::WeightInfo::authorize_call())]
+		#[pallet::weight((
+			T::WeightInfo::authorize_call(),
+			DispatchClass::Operational,
+			Pays::No,
```
Comment (xlc) [inline authority/src/lib.rs:348]: I am not 100% sure if this will still charge fee if this call failed. Can you verify?

Comment (ermalkaleci) [reply authority/src/lib.rs:348]: ok

Comment (xlc) [reply authority/src/lib.rs:348]: I know it is safe to do like this https://github.com/open-web3-stack/open-runtime-module-library/blob/69426e8597ebc7c7922b81dbb2f6cc2ab7851e86/oracle/src/lib.rs#L165
An error will not return Pays::No.

Comment (ermalkaleci) [reply authority/src/lib.rs:348]: @xlc yes it will

Comment (ermalkaleci) [reply authority/src/lib.rs:348]: Post dispatch info will be `{ post_info: PostDispatchInfo { actual_weight: None, pays_fee: Pays::Yes }`

**Review by xlc — APPROVED** [review]

**Pull request closed without merging**